var/spack/repos/builtin/packages/py-pylint/package.py

```diff
@@ -38,20 +40,23 @@ class PyPylint(PythonPackage):
     depends_on('py-astroid@2.2.0:2', type=('build', 'run'), when='@2.3.0:2.7')
     depends_on('py-astroid@2.5.6:2.6', type=('build', 'run'), when='@2.8.0:2.10')
     depends_on('py-astroid@2.8.0:2.8', type=('build', 'run'), when='@2.11.1:')
```

Comment:
```suggestion
    depends_on('py-astroid@2.8.0:2.8', type=('build', 'run'), when='@2.11.1')
```
Otherwise this constraint will also be applied to 2.13.5 and it won't be possible to install. Ideally you could figure out what version this dependency changed in but that isn't required.